src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -1910,9 +1926,13 @@ public Vector3 WorldScale
             get
             {
                 Vector3 temp = new Vector3(0.0f, 0.0f, 0.0f);
+#if NUI_PROPERTY_CHANGE_1
+                Object.InternalRetrievingPropertyVector3(SwigCPtr, View.Property.WorldScale, temp.SwigCPtr);
```

Comment:
same as above

Reply:
I found that `WorldScale` is exactly equal with `CurrentWorldScale` intenrally. So I make it `ViewInternal.cs` API, which we already changed.